services/core-api/app/api/mines/permits/permit_extraction/create_permit_conditions.py

```diff
@@ -0,0 +1,162 @@
+import uuid
```

Comment:
This file contains pretty much all the logic for mapping between conditions returned from the permit service to the `PermitConditions` model in Core, including mapping of Category, and building of a nested structure